### ISSUE TITLE
File navigation issue when using Windows Host and Linux Remote

### ISSUE THREAD

**Issue by tvvignesh** [Huachao/vscode-restclient#715]
- REST Client Version: 0.24.4
- VSCode Version:
```
Version: 1.52.0-insider (user setup)
Commit: 050a123750e4ef4009930d0da1ff7a1a6441d4f3
Date: 2020-11-12T05:43:04.852Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.20201
```
- OS Version: Windows 10 Home Insiders (Host), Ubuntu 20.04 (Remote)

Steps to Reproduce:

Hi. I use the extension with Windows Host and Linux remote via remote ssh. While it works, clicking this to do file navigation fails:

<img width="224" alt="2" src="https://user-images.githubusercontent.com/1165845/99086347-65c8c500-25ef-11eb-9da1-e59f12b9f987.PNG">

Probably since there is a difference in path interpretation in VSCode (not sure). I get this error:

<img width="259" alt="1" src="https://user-images.githubusercontent.com/1165845/99086424-855fed80-25ef-11eb-9594-c29b316d5e07.png">

I guess this might help: https://github.com/microsoft/vscode-remote-release/issues/585

**Comment by tvvignesh** [comment]
While not related to this issue, another suggestion I have is to support autocomplete of file paths when typing it to load.

**Comment by Huachao** [comment]
@tvvignesh do you mean that the outgoing request body is correct indeed, while the go to document link doesn't work?

**Comment by tvvignesh** [comment]
@Huachao Yup. The request works well and the request body is correct (The file does exist in the path). Only the navigation to file when doing `Ctrl+Click` is broken when using is in Remote SSH with Windows Host and Linux remote

**Comment by Huachao** [comment]
@tvvignesh could you please also provide the absolute file path of the executed HTTP file and your actual absolute file path of the referenced gql file?

**Comment by tvvignesh** [comment]
@Huachao Hi. Just tried with the absolute path instead of relative path. Have the same issue but the request works.

<img width="238" alt="1" src="https://user-images.githubusercontent.com/1165845/99213471-c0cb0980-27f3-11eb-9287-5611f9a58236.png">

**Comment by tvvignesh** [comment]
The problem is not about the absolute or relative path here. The problem is that it is trying to access the host machine instead of the remote machine. I tested it out and found that `/` is pointing to the root of my host (C: drive in my case), not the remote Linux host. And the same is the case with relative path as well.

**Comment by Shawn1874** [comment]
Interesting issue.  Does the send button work for you during a remote explorer connection to the Linux VM, because it doesn't show within my text editor when I have a .rest file opened while connected to a RHEL8 Linux server.
![image](https://github.com/Huachao/vscode-restclient/assets/18298395/9ff283b8-737d-4237-a64b-24fa34382647)
